src/scripts.js

```diff
@@ -45,14 +44,46 @@ const homeButton = document.querySelector('.home-button');
 const searchBar = document.getElementById('searchInput');
 const filterTags = document.querySelectorAll('.tag');
 const favoritePageButton = document.getElementById('favoritesPage');
+const addToCookbookButton = document.querySelector('.cookbook-button-js')
+const cookbookPageButton = document.querySelector('.cookbook-page-button-js');
+
 
 // EVENT LISTENERS
-searchRecipesButton.addEventListener('click', searchAllRecipes)
+searchRecipesButton.addEventListener('click', searchAllRecipes);
 filterButton.addEventListener('click', filterAllRecipesByTag);
 homeButton.addEventListener('click', displayHomePage);
 favoritePageButton.addEventListener('click', displayFavorites);
+addToCookbookButton.addEventListener('click', toggleCookbookButton);
+cookbookPageButton.addEventListener('click', displayCookbook);
 
 // FUNCTIONS
+function toggleCookbookButton() {
```

Comment:
Will ask Hannah tomorrow if a better practice is to split this up- essentially have two helper functions holding all these lines of code and then invoke them inside this function.